### ISSUE TITLE
New Layout components implemented across Polaris

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris-react-archive#5615]
**TLDR;** Create Layout primitives, remove custom css and apply them to Polaris components to increase mainline coverage by +14%.

Layout is the biggest factor today for lack of mainline coverage. Product teams building new layouts are restricted by Polaris and often create custom solutions. Polaris only provides two layout components which aren’t always the best solution for a specific problem. Even inside the Polaris components we - the Polaris team - only use our own layout components 20% of the time.

Creating new layout components and applying them to the existing Polaris components will allow us to create a flexible system. These layout components will enable product teams to create more intentional layouts with minimal tweaks while staying on mainline.

**Comment by Shefiuecomms** [comment]
Yes, I will help on it

**Comment by alex-page** [comment]
Hey @Shefiuecomms we are already underway. If you want to test them please do the components are live as `alpha` components on polaris.shopify.com.

**Comment by sam-b-rose** [comment]
Closing this one out. The project has stopped and may resume along side other breaking change component API improvements
